.nvmrc

```diff
@@ -1 +1 @@
-v8.1.0
+9.11.1
```

Comment:
shouldn't it be ignored?

Reply:
No, `.nvmrc` defines which version should be used when running the project. Of course it can be ignored though. I couldn't compile with <9.0 with yarn so I updated to 9.11.1